app/controllers/users_controller.rb

```diff
@@ -183,7 +194,7 @@ def fail_with(key)
 
 
   def user_params
-    params.permit(:email, :password, :firstname, :lastname, :phone, :status)
+    params.permit(:email, :password, :first_name, :last_name, :phone, :status)
```

Comment:
@vinomca-megam  Can you check if the `first_name` and `last_name` works in `createAccount`  ?